.github/workflows/build-and-push-binaries.yml

```diff
@@ -104,7 +121,7 @@ jobs:
           git config --global user.email "no-reply@konveyor.io"
           git config --global user.name "Konveyor CI"
           git clone https://github.com/konveyor/rulesets
```

Comment:
Do we want to use the `make` targets for the CI?

Reply:
yes, but also followup?

Reply:
Sure thing, I'll make an issue.

Reply:
#520